gui/wxpython/gui_core/mapdisp.py

```diff
@@ -721,3 +685,39 @@ def OnDraw(self, event):
     def Draw(self, mapToDraw):
         """Re-display current map composition"""
         mapToDraw.UpdateMap(render=False)
+
+
+class FrameMixin:
+    """Mixin class for wx.Panel that provides methods standardly
+    used on wx.Frame widget"""
+
+    def Show(self):
+        self.GetParent().Show()
+
+    def SetTitle(self, name):
+        self.GetParent().SetTitle(name)
+
+    def Raise(self):
+        self.GetParent().Raise()
+
+    def SetFocus(self):
+        self.GetParent().SetFocus()
+
+    def OnFullScreen(self, toolbars, auimgr, event):
```

Comment:
I think this should be `def ShowFullScreen(self, show)` to resemble the actual TopLevelWindow method and inside you would hide/show the toolbars and I think also statusbar. Method IsFullScreen should be defined outside of this method.

Reply:
Coded :-)